src/blocks/blocks/button-group/button/inspector.js

```diff
@@ -301,7 +301,7 @@ const Inspector = ({
 												label={ __( 'Background', 'otter-blocks' ) }
 												colorValue={ attributes.background }
 												gradientValue={ attributes.backgroundGradient }
-												onColorChange={ background => setAttributes({ background: background })}
+												onColorChange={ background => setAttributes({ background })}
```

Comment:
Not sure if this was automated but we don't need it. If it was automated using eslint then it's fine.

Reply:
Removing this as I read it the other way. 😄